bigquery/samples/tests/test_table_samples.py

```diff
@@ -15,11 +15,10 @@
 from .. import create_table
 
 
-def test_table_samples(capsys, client, project_id, dataset_id, table_id):
-    """Since creating a table is a long operation, test all table model samples
+def test_table_samples(capsys, client, random_table_id):
+    """Since creating a table is a long operation, test all table samples
```

Comment:
Creating a table isn't all that long of an operation. You can test each sample separately. This docstring comment can be removed.

Reply:
Removed this docstring.